Releasenotes.md

```diff
@@ -0,0 +1,77 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-hitorie-bevragen API als release is uitgebracht zijn de specificaties van deze enspoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar redelijk stabiel geweest.
+Voor het uitbrengen van release v1.0.0 is besloten om de historie-endpoints in een aparte API onder te brengen en daarvoor een eigen github-repository te maken. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Dit is een globaal (en niet limitatief) overzicht van wijzigingen die doorgevoerd zijn voordat v1.0.0 is uitgebracht. Enkele van deze wijzigingen zijn breaking ten opzichte van v0.9.0.
+
+We bieden nu ook SDK's aan met gegenereerde [plumbing-code](./code). Daarnaast bieden we een[postman-collectie](./test) t.b.v. testen aan.
+
+### Openapi.yaml :
+
+- server-url is aangepast (omdat het een aparte API is geworden.) --> https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen _**(breaking)**_
+- OperationId's zijn aangepast ivm code-generatie issues en in nu in UpperCamelCase. _**(breaking)**_
+- Alle descriptions zijn omgezet naar markdown.
+- Bij properties zijn de maxLength, minLength, pattern en (waar overbodig) de title weggehaald.
+- Descriptions zijn aangepast om beter bij te gebruikersbeleving aan te sluiten, overbodige descriptions zijn verwijderd.
+- Enkele parameternamen zijn aangepast vanwege consistentie met andere Haal-Centraal API's en/of de Design Decision om parameternamen in camelCase op te nemen  
```

Comment:
ik wil er voor pleiten de belangrijkste wijzigingen, in dit geval de breaking changes, bovenaan te zetten. En de documentatiewijzigingen onderaan.